tests-scan

```diff
@@ -462,7 +401,7 @@ def scan_for_pull_tasks(api, contexts, opts, repo):
         results.sort(reverse=True, key=str)
         return [tests_human(job, options=opts) for job in results]
     if not opts.amqp:
-        return [tests_invoke(job, options=opts) for job in results]
+        return [queue_test(job, None, options=opts) for job in results]
```

Comment:
Note: This isn't super-nice or elegant. But this beast is due for a rewrite anyway, I just want to shed some dead weight first.